lib/tizen_device.dart

```diff
@@ -267,7 +267,14 @@ class TizenDevice extends Device {
       }
     }
     _logger.printTrace('Installing TPK.');
-    if (!await installApp(package)) {
+    if (await installApp(package)) {
+      // On TV emulator, a tpk must be installed twice if it's being installed
+      // for the first time in order to prevent a library loading error.
+      // Issue: https://github.com/flutter-tizen/flutter-tizen/issues/50
+      if (!wasInstalled && usesSecureProtocol && await isLocalEmulator) {
+        await installApp(package);
```

Comment:
Interesting and it's hard to understand easily. does this solve the loading problem that occur when launching fresh app?

Reply:
Yeah, it's a hack and I don't understand why it works either. We may have to contact the TV SDK team if you think this is not a right way to resolve the issue.

Reply:
I think it's gonna take a lot of time. Let's just keep moving on